src/main/resources/application.yaml

```diff
@@ -1,4 +1,6 @@
 spring:
+  messages:
+    encoding: UTF-8
```

Comment:
Doesn't seem to affect downstream apps - they will use ISO as default for .properties files